lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -286,23 +350,33 @@ class RenderPopover extends RenderShiftedBox {
       arrowCenter = 0.5;
     }
 
-    final borderPath = _buildBorderPath(direction, arrowCenter);
+    final backgroundShapePath = _buildBackgroundShapePath(direction, arrowCenter);
+
+    // Cache the background shape path, so it can be used in hit-testing.
+    _backgroundShapePath = backgroundShapePath;
```

Comment:
Why don't you just use `_backgroundShapePath` instead of an intermediate variable?